source/documentation/index.md

```diff
@@ -75,6 +75,10 @@ The [API changelog](https://github.com/hmrc/income-tax-mtd-changelog) (GitHub) p
 
 Below is a summary of updates to this service guide.
 
+#### 24 March 2025
+
+Update [Amendments after making a final declaration](/guides/income-tax-mtd-end-to-end-service-guide/documentation/make-updates-during-tax-year.html#amendments-after-making-a-final-declaration)
```

Comment:
I don't think this is the correct url, I think it should be `/guides/income-tax-mtd-end-to-end-service-guide/documentation/make-updates-at-tax-year-end.html#amendments-after-making-a-final-declaration` i.e. making updates at tax year end. This doesn't link to the correct section but this does https://developer.qa.tax.service.gov.uk/guides/income-tax-mtd-end-to-end-service-guide/documentation/make-updates-at-tax-year-end.html#amendments-after-making-a-final-declaration